src/platform/apollolake/include/platform/memory.h

```diff
@@ -182,14 +188,16 @@
 
 #define HEAP_RUNTIME_SIZE \
 	(HEAP_RT_COUNT64 * 64 + HEAP_RT_COUNT128 * 128 + \
-	HEAP_RT_COUNT256 * 256 + HEAP_RT_COUNT512 * 512 + \
-	HEAP_RT_COUNT1024 * 1024)
+	HEAP_RT_COUNT192 * 192 + HEAP_RT_COUNT256 * 256 + \
+	HEAP_RT_COUNT320 * 320)
 
 #define HEAP_BUFFER_BASE	(HEAP_RUNTIME_BASE + HEAP_RUNTIME_SIZE)
 #define HEAP_BUFFER_SIZE	(SOF_STACK_END - HEAP_BUFFER_BASE)
 #define HEAP_BUFFER_BLOCK_SIZE	0x180
 #define HEAP_BUFFER_COUNT	(HEAP_BUFFER_SIZE / HEAP_BUFFER_BLOCK_SIZE)
 
+#define PLATFORM_BUFFER_BLOCK_SIZE (HEAP_BUFFER_BLOCK_SIZE - 64)
```

Comment:
I don't understand this.

Reply:
@tlauda , please ignore this for I may rework the PR since The memory structure is significant different with the one on which I made this PR based